rust/rope/src/engine.rs

```diff
@@ -33,7 +33,7 @@ pub struct Engine {
 
 struct Revision {
     rev_id: usize,
-    from_union: Subset,
+    deletes_from_union: Subset,
```

Comment:
It was a totally fine name back when `Subset` actually was supposed to represent a transformation of a document to a subset of it, then the type told you what it was. Under the new model though, it needs a longer name to be unambiguous.

Also, Sublime has really good auto-complete for terms that are used frequently within a single file, so for me it isn't any more difficult to type. Everyone else should get a better editor, although I know they're working on it. 😉